Classes/Plugins/Ads/PKAdInfo.swift

```diff
@@ -46,21 +47,6 @@ import GoogleInteractiveMediaAds
         }
     }
     
-    init(ad: IMAAd) {
```

Comment:
the issue now uses the AdStarted event instead (in future we will need to separate them again).
Docs are updated.